build/status/Nl92aXpwcmVwL291dC9yaXZlcnNfZnVuLnJkcy5pbmQ.yml

```diff
@@ -0,0 +1,13 @@
+version: 0.3.0
+name: 6_vizprep/out/rivers_fun.rds.ind
+type: file
+hash: ca0e1e89f96e3012726138f15d77a357
+time: 2018-07-12 01:14:00 UTC
+depends:
+  1_fetch/out/river_geometry.rds.ind: 0afc7326cf6474c9836b879d206aa545
+fixed: 02a0352cba4af48ec5e70e0ebc354ea1
+code:
+  functions:
+    prep_rivers_fun: dfc3854f0aea696b9e6704849547c2f4
+    write_put_fun: a1f9c3688ca2f519a05753c52f90957e
```

Comment:
can this build/status file be deleted, because rivers_fun.rds.ind is now just a rivers_fun R object?